docs/resources/federation_federated_block.md

```diff
@@ -0,0 +1,67 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_federation_federated_block Resource - terraform-provider-bloxone"
+subcategory: ""
+description: |-
+  
+---
+
+# bloxone_federation_federated_block (Resource)
+
+
+
+## Example Usage
+
+```terraform
+resource "bloxone_federation_federated_realm" "example" {
+  name = "example_federation_federated_realm"
+}
+
+resource "bloxone_federation_federated_block" "example" {
```

Comment:
does this also support next available ?

Reply:
Yes, it supports the next available federated block, and we are not implementing the support as part of this epic.